tests/tt_eager/python_api_testing/sweep_tests/test_configs/ci_sweep_tests_working/grayskull/pytorch_transpose_cn_test.yaml

```diff
@@ -18,7 +18,7 @@ test-list:
           low: -100
           high: 100
       comparison:
-        function: comp_equal
```

Comment:
These tms should be able to pass with comp_equal if the input tensor was created with bfloat16 right?

Reply:
Yes, I will update this as well